### DISCUSSION TITLE
Don't see any Nibe devices in my Homebridge

### DISCUSSION THREAD

**Discussion by konki-vienna** [hp-net/homebridge-nibe#232] [Q&A]
Hi,
I have a NIBE S1255-6 heat pump and a paid myUplink account. I wanted to connect it via your plugin to homebridge to read some of the data. I followed the setup instructions (adding Client Identifier and Client Secret), but I do not see any new devices in my homebridge instance.
Any advise what to do? I run the latest version of homebridge and the latest version of your plugin (v.2.0.1)

**Comment by hpruszyn** [comment]
Discussion moved to #234
